### PR TITLE
[SPARK-28791] [DOC] Documentation for Alter table Command

### PR DESCRIPTION
**Pull request by PavithraRamachandran**
What changes were proposed in this pull request?
Document ALTER TABLE statement in SQL Reference Guide.

Why are the changes needed?
Adding documentation for SQL reference.

Does this PR introduce any user-facing change?
yes

Before:
There was no documentation for this.

After.

How was this patch tested?
Used jekyll build and serve to verify.

### CODE REVIEW

**Comment by dilipbiswal** [comment]
@PavithraRamachandran Could you please re-check this PR ? It seems to contain un-related changes.

**Comment by gatorsmile** [comment]
ok to test

**Comment by PavithraRamachandran** [comment]
@dilipbiswal could u review the document?

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 1]
```diff
@@ -19,4 +19,239 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+`ALTER TABLE` statement changes the schema or properties of a table.
+
+### RENAME 
+`ALTER TABLE RENAME` statement changes the table name of an exsisting table in the Database.
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: Database -> database.
You might need a blank line after this line.

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 2]
```diff
@@ -19,4 +19,239 @@ license: |
...
+<dl>
+  <dt><code><em>COLUMNS (col_spec)</em></code></dt>
+  <dd>Specifies the columns to be added to be renamed.</dd>
+</dl>
+
+
+### SET AND UNSET
+
+#### SET TABLE PROPERTIES
+`ALTER TABLE SET` command is used for setting the Table properties.If a particular property was already set,
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: Table -> table
space after period

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 3]
```diff
@@ -19,4 +19,239 @@ license: |
...
+</dl>
+
+
+### SET AND UNSET
+
+#### SET TABLE PROPERTIES
+`ALTER TABLE SET` command is used for setting the Table properties.If a particular property was already set, 
+this overrides the old value with the new one.
+
+`ALTER TABLE USET` is used to drop the table Property.
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: `UNSET`
Property -> property

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 4]
```diff
@@ -19,4 +19,239 @@ license: |
...
+--Set Table Properties
+ALTER TABLE table_name SET TBLPROPERTIES (key1=val1, key2=val2, ...)
+
+--Unset Table Properties
+ALTER TABLE table_name UNSET TBLPROPERTIES [IF EXISTS] (key1, key2, ...)
+
+{% endhighlight %}
+
+#### SET SERDE
+`ALTER TABLE SET` command is used for setting the SERDE or SERDE Propeties in Hive tables.If a particular property was already set,
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: Space after period

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 5]
```diff
@@ -19,4 +19,239 @@ license: |
...
+  <dt><code><em>SERDEPROPERTIES (key1=val1, key2=val2, ...)</em></code></dt>
+  <dd>Specifices the SERDE properties to be set.</dd>
+</dl>
+
+
+### Examples
+{% highlight sql %}
+
+--RENAME table 
+desc student;
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: DESC

**Review by srowen — CHANGES_REQUESTED** [review]

**Comment by PavithraRamachandran** [comment]
@srowen i have addressed the review comments could you review.

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 6]
```diff
@@ -19,4 +19,240 @@ license: |
...
+| rollno                   | int        | NULL     |
+| age                      | int        | NULL     |
+| # Partition Information  |            |          |
+| # col_name               | data_type  | comment  |
+| age                      | int        | NULL     |
++--------------------------+------------+----------+--+
+
+--RENAME partition
+
+show partitions StudentInfo;
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: Let's consistently UPPER CASE reserved words like `SHOW PARTITIONS`

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 7]
```diff
@@ -19,4 +19,240 @@ license: |
...
+{% endhighlight %}
+
+#### Parameters
+<dl>
+  <dt><code><em>old_table_name</em></code></dt>
+  <dd>Name of an existing table.</dd>
+</dl>
+<dl>
+  <dt><code><em>db_name</em></code></dt>
+  <dd>Name of the  exsisting database.</dd>
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: Small nit: extra space here

**Comment by PavithraRamachandran** [comment]
@srowen reworked on the review comments. Could you review.

**Comment by srowen** [comment]
@PavithraRamachandran did you push?

**Comment by PavithraRamachandran** [comment]
@srowen i have now..

File: docs/sql-ref-syntax-ddl-alter-table.md [hunk 8]
```diff
@@ -19,4 +19,240 @@ license: |
...
+</dl>
+
+
+### SET AND UNSET
+
+#### SET TABLE PROPERTIES
+`ALTER TABLE SET` command is used for setting the table properties. If a particular property was already set, 
+this overrides the old value with the new one.
+
+`ALTER TABLE USET` is used to drop the table property.
```
Comment (srowen) [inline docs/sql-ref-syntax-ddl-alter-table.md]: There are still some issues from the previous review like this one: `UNSET`

**Comment by PavithraRamachandran** [comment]
@srowen sorry..I missed that. I have corrected . Could you review

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**